app/styles/tale-tabs.scss

```diff
@@ -38,6 +38,6 @@
 }
 
 #taleMetadataTab {
-  overflow-x: auto;
-  max-height: 50vh;
+  overflow: auto;
+  height: calc(100vh - (2.5rem + 50px + 40px + 50px + 0px + 94px + 50px));
```

Comment:
Please, use the variables instead, so we just need to change a single value and it trickles down to the whole app if a height changes for some of the other elements, e.g: `height: calc(100vh - (#{$bottom-separation} + #{$inverted-menu-height} + #{$main-menu-height} + #{$footer-height} + #{$wt-paddleboard-panel-height} + #{$wt-paddleboard-panel-tall-height}));`

Reply:
This is the current status:
<img width="1358" alt="screen shot 2018-12-07 at 2 05 44 pm" src="https://user-images.githubusercontent.com/20361247/49667495-3c798480-fa29-11e8-84db-8a501fbbb6bf.png">